camayoc/api.py

```diff
@@ -226,4 +201,8 @@ def request(self, method, url, **kwargs):
         #
         #     request(method, url, **kwargs)
         #
+        headers = self.default_headers()
+        headers.update(kwargs.get('headers', {}))
+        kwargs['headers'] = headers
+        kwargs.setdefault('verify', self.verify)
```

Comment:
This is good, because for the facts and reports we will have to set the media type we will accept in the header